pkg/build/generator/generator_test.go

```diff
@@ -43,15 +43,15 @@ const (
 )
 
 func TestInstantiate(t *testing.T) {
-	generator := mockBuildGenerator(nil, nil, nil, nil, nil, nil, nil)
+	generator := mockBuildGenerator(&TestingClient{})
```

Comment:
you *could* be nice and allow the caller to pass a nil TestingClient and create one for them inside mockBuildGenerator....